lib/js/components/Modals/Modal/Modal.stories.ts

```diff
@@ -0,0 +1,150 @@
+		};
+	},
+});
+
+export const Interactive = StoryTemplate.bind({});
+
+const args = {
+	size: MODAL_SIZES.MEDIUM,
+	danger: false,
+	contentCentered: false,
+	headerImage: '',
+	headerTitleSize: MODAL_HEADER_TITLE_SIZES.MEDIUM,
+	headerTitle: 'Modal Title Accusantium occaecati atque similique gg provident nobis.',
+	headerSubtitle: 'Modal Subtitle Aut vitae neque consequatur. Consequuntur natus sint ad.',
+	headerFeatureIcon: 'FA_CIRCLE_EXCLAMATION',
+	headerFeatureIconColor: FEATURE_ICON_COLOR.NEUTRAL,
+	footerPrimaryButtonText: 'Primary M',
+	footerPrimaryButtonIcon: null,
+	footerSecondaryButtonText: 'Secondary M',
+	footerSecondaryButtonIcon: null,
+	footerTertiaryButtonText: 'Tertiary M',
+	footerTertiaryButtonIcon: null,
+	footerCheckboxText: '',
+} as Args;
+
+const argTypes = {
+	size: {
+		control: { type: 'select', options: Object.values(MODAL_SIZES) },
+		defaultValue: MODAL_SIZES.MEDIUM,
+	},
+	danger: { control: { type: 'boolean' }, defaultValue: false },
+	headerTitleSize: {
+		control: { type: 'select', options: Object.values(MODAL_HEADER_TITLE_SIZES) },
+		defaultValue: MODAL_SIZES.MEDIUM,
+	},
+	headerTitle: { control: { type: 'text' } },
+	headerImage: { control: { type: 'text' } },
+	headerSubtitle: { control: { type: 'text' } },
+	headerFeatureIcon: {
+		control: { type: 'select', options: [null, ...Object.keys(ICONS)] },
+		defaultValue: 'FA_CIRCLE_QUESTION',
+	},
+	headerFeatureIconColor: {
+		control: { type: 'select', options: Object.values(FEATURE_ICON_COLOR) },
+		defaultValue: FEATURE_ICON_COLOR.NEUTRAL,
+	},
+	contentCentered: { control: { type: 'boolean' }, defaultValue: false },
+	footerPrimaryButtonText: { control: { type: 'text' } },
+	footerPrimaryButtonIcon: {
+		control: { type: 'select', options: [null, ...Object.keys(ICONS)] },
+		defaultValue: null,
+	},
+	footerSecondaryButtonText: { control: { type: 'text' } },
+	footerSecondaryButtonIcon: {
+		control: { type: 'select', options: [null, ...Object.keys(ICONS)] },
+		defaultValue: null,
+	},
+	footerTertiaryButtonText: { control: { type: 'text' } },
+	footerTertiaryButtonIcon: {
+		control: { type: 'select', options: [null, ...Object.keys(ICONS)] },
+		defaultValue: 'FA_CHART_COLUMN',
+	},
+	footerCheckboxText: { control: { type: 'text' } },
+} as ArgTypes;
+
+Interactive.argTypes = argTypes;
+Interactive.args = args;
+
+Interactive.parameters = {
+	design: {
+		type: 'figma',
+		url: 'https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS-Components?node-id=5658%3A96416&t=lgUXCZhvWDuBr3qw-0',
+	},
+};
+
+const argsDanger = {
+	danger: true,
+	headerImage: '',
+	headerTitle: 'Modal Title Accusantium occaecati atque similique gg provident nobis.',
+	headerSubtitle: 'Modal Subtitle Aut vitae neque consequatur. Consequuntur natus sint ad.',
+	slot: 'Text. Occaecati qui debitis ut minima ab ut ipsa. Eos qui eum. Vel eaque sint. Aut voluptatem est. Eos est aut sit soluta est facere qui soluta aspernatur. Et sint repellat sunt.',
```

Comment:
do we use it anywhere?

Reply:
I refactored it to `defaultSlot` and use this variable in the html template.